### ISSUE TITLE
Not working on Tuinity and Akarin

### ISSUE THREAD

**Issue by satyrnhub** [SkriptLang/Skript#3072]
### Description
Skript doesn't load on Tuinity or Akarin.

### Steps to Reproduce
Install either Tuinity or Akarin.
Install Skript 2.5-alpha3 or 2.4.1.
And, it fails to load.

### Expected Behavior
Skript would load as normal and all the commands would work as expected.

### Errors / Screenshots
https://pastebin.com/QzV2asDD

### Server Information
* **Server version/platform:**       git-Tuinity-"1aa9097" / git-Akarin-"b7c5442"
* **Skript version:**                2.5-alpha3 / 2.4.1

**Comment by Wealthyturtle** [comment]
I'm quite sure some people have used Tuinity with no problems on Skript (plus, your error log only shows Akarin).
At the moment, Skript only supports Spigot/Paper, and there's no official support for forks of those, so you may want to take it up with the Akarin devs.

**Comment by satyrnhub** [comment]
I already did contact them, and I have tried Tuinity with the same issue.

**Comment by satyrnhub** [comment]
Can you please reopen this? An Akarin dev redirected me here.

**Comment by ShaneBeee** [comment]
Just tested with Tunitiy, result:
```
0:31:24 INFO]: [Skript] Enabling Skript v2.5-alpha3
[20:31:26 INFO]: [Skript] You're currently running the latest stable version of Skript.
[20:31:29 INFO]: [Skript] Loaded 114752 aliases in 4059ms
[20:31:29 INFO]: [Skript]  ~ created by & © Peter Güttinger aka Njol ~
~~~~~
[20:31:30 INFO]: [Skript] Loading variables...
[20:31:30 INFO]: [Skript] Loaded 0 variables in 0.0 seconds
[20:31:30 INFO]: [Skript] Loaded 1 script with a total of 1 trigger and 0 commands in 0.05 seconds
[20:31:30 INFO]: [Skript] Finished loading.
[20:31:30 INFO]: Done (15.367s)! For help, type "help"
```

Info:
```
> ver skript
[20:31:47 INFO]: Skript version 2.5-alpha3
[20:31:47 INFO]: Customize Minecraft's mechanics with simple scripts written in plain English sentences.
[20:31:47 INFO]: Website: https://skriptlang.github.io/Skript
[20:31:47 INFO]: Authors: Njol, Mirreski, bensku, TheBentoBox, FranKusmiruk, ShaneBeee, Blueyescat, JRoy, Nicofisi, APickledWalrus, xXAndrew28Xx, Syst3ms, TheLimeGlass, Pikachu920, Sashie, Wealthyturtle, OfficialDonut and eyesniper2
> ver
[20:31:50 INFO]: This server is running Tuinity version git-Tuinity-"98e9d5b" (MC: 1.15.2) (Implementing API version 1.15.2-R0.1-SNAPSHOT)
[20:31:50 INFO]: Checking version, please wait...
[20:31:51 INFO]: Previous version: git-Tuinity-"e5045a0" (MC: 1.15.2)
[20:31:51 INFO]: You are running the latest version
```

Needless to say, I can not replicate this issue.

**Comment by ShaneBeee** [comment]
Tested on Akarin:
```
20:37:10 INFO]: [Skript] Enabling Skript v2.5-alpha3
[20:37:12 INFO]: [Skript] You're currently running the latest stable version of Skript.
[20:37:15 INFO]: [Skript] Loaded 114752 aliases in 3821ms
[20:37:15 INFO]: [Skript]  ~ created by & © Peter Güttinger aka Njol ~
~~~~
[20:37:16 INFO]: [Skript] Loading variables...
[20:37:16 INFO]: [Skript] Loaded 0 variables in 0.0 seconds
[20:37:16 WARN]: [Skript] 'test.sk' is empty
[20:37:16 INFO]: [Skript] All scripts loaded without errors.
[20:37:16 INFO]: [Skript] Loaded 1 script with a total of 0 triggers and 0 commands in 0.14 seconds
[20:37:16 INFO]: [Skript] Finished loading.
[20:37:16 INFO]: Done (14.623s)! For help, type "help"
```

Info:
```
> ver skript
[20:37:52 INFO]: Skript version 2.5-alpha3
[20:37:52 INFO]: Customize Minecraft's mechanics with simple scripts written in plain English sentences.
[20:37:52 INFO]: Website: https://skriptlang.github.io/Skript
[20:37:52 INFO]: Authors: Njol, Mirreski, bensku, TheBentoBox, FranKusmiruk, ShaneBeee, Blueyescat, JRoy, Nicofisi, APickledWalrus, xXAndrew28Xx, Syst3ms, TheLimeGlass, Pikachu920, Sashie, Wealthyturtle, OfficialDonut and eyesniper2
> ver
[20:37:53 INFO]: This server is running Akarin version git-Akarin-"b7c5442" (MC: 1.15.2) (Implementing API version 1.15.2-R0.1-SNAPSHOT)
[20:37:53 INFO]: Checking version, please wait...
[20:37:54 INFO]: Previous version: git-Tuinity-"98e9d5b" (MC: 1.15.2)
[20:37:54 INFO]: Unknown version
```

Again, can not replicate this issue.
Its hard to fix something we can not replicate.

**Comment by ShaneBeee** [comment]
To add to this, as WeathlyTurtle said, we support Spigot and Paper, the two most used server softwares for Minecraft.
We can't possibly add full support for every fork out there.
That said, if Skript works on Paper, but not one of its forks, that means of the forks changed something that somehow breaks Skript.
In my testing of both those forks, they both loaded without any issues/errors. 
As stated, we can't not fix an issue we can't find.

**Comment by josephworks** [comment]
@ShaneBeee thank you for your tests. I understand that Skript does not support paper forks, but through what you shared with your tests on akarin, I conclude the issue is from @satyrnhub 's skript configs. Sorry for the inconvenience, I've closed the issue on Akarin as well.

Keep up the great work here, and thank you for all of your works.

**Comment by satyrnhub** [comment]
Agreed. Thank you for your help!

**Comment by FranKusmiruk** [comment]
For the record, it is true that we don't support anything besides Spigot and Paper, but that doesn't mean we are not willing to fix compatibility issues with other forks as long as they don't fundamentally change the APIs used by Skript. We have done this for things such as FAWE and WG, don't see why it wouldn't be the case for projects like Akarin.

**Comment by josephworks** [comment]
Good to know! Tuinity and Akarin haven't made any changes to the Paper API and I doubt that either are planning to do so. Most of our work is modifying the server, not API.
